src/spi_flash.c

```diff
@@ -179,13 +179,13 @@ uint16_t spi_flash_probe(void)
     spi_write(0xFF);
     product = spi_read();
     spi_cs_off(SPI_CS_FLASH);
-    if (manuf == 0xBF)
+    if (manuf == 0xBF || manuf == 0xC2)
         chip_write_mode = SST_SINGLEBYTE;
     if (manuf == 0xEF)
         chip_write_mode = WB_WRITEPAGE;
 
 #ifndef READONLY
-    spi_cmd(EWSR);
+    //spi_cmd(EWSR); // TODO not supported on every flash, but necessary for some models?
```

Comment:
I have tested this change with SST25VF080B. The line is in fact not necessary and can be safely removed. Please remove the comment.